spec/views/layouts/listnav/_ems_cloud.html.haml_spec.rb

```diff
@@ -10,11 +10,11 @@
   end
 
   it "Flavors link for Openstack cloud manager uses restful path" do
-    record =  ManageIQ::Providers::Openstack::CloudManager.new(:name => "Test Cloud")
+    record = FactoryGirl.create(:ems_openstack, :id => 10)
```

Comment:
Note that since you are verifying the response string there you may need the compressed id (only apparent on non-0 regions, which on Travis we use region 1), in which case you can call `record.compressed_id`